File: app/src/main/java/com/callmangement/model/reports/Monthly_Reports_Info.java
package com.callmangement.model.reports;

import java.io.Serializable;

public class Monthly_Reports_Info implements Serializable{
    private String date;
    private int not_resolved;
    private int resolved;
    private int total;

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public int getNot_resolved() {
        return not_resolved;
    }

    public void setNot_resolved(int not_resolved) {
        this.not_resolved = not_resolved;
    }

    public int getResolved() {
        return resolved;
    }

    public void setResolved(int resolved) {
        this.resolved = resolved;
    }

    public int getTotal() {
        return total;
    }

    public void setTotal(int total) {
        this.total = total;
    }
}

File: app/src/main/java/com/callmangement/model/reports/Monthly_Reports_Info.kt
package com.callmangement.model.reports

import java.io.Serializable

class Monthly_Reports_Info : Serializable {
    @JvmField
    var date: String? = null
    @JvmField
    var not_resolved: Int = 0
    @JvmField
    var resolved: Int = 0
    @JvmField
    var total: Int = 0
}
